pkg/cloudprovider/types.go

```diff
@@ -94,16 +97,19 @@ type Packing struct {
 type TopologyKey string
 
 const (
-	TopologyKeyZone   TopologyKey = "zone"
-	TopologyKeySubnet TopologyKey = "subnet"
+	TopologyKeyZone TopologyKey = "zone"
 )
 
-// Architecture constrains the underlying node's compilation architecture.
 type Architecture string
 
 const (
-	ArchitectureLinux386 Architecture = "linux/386"
-	// LinuxAMD64 Architecture = "linux/amd64" TODO
+	ArchitectureAmd64 Architecture = "amd64"
```

Comment:
Would be great to get `aarch64` in here too. (I realize that's probably a separate commit, mostly just want to remind myself).